docs/artifact_repositories/artifactory.md

```diff
@@ -0,0 +1,75 @@
+# Artifactory
+
+Artifactory can handle both local and remote repositories. Local registries can
+store and serve your internal packages; whereas, remote repositories will pull
+packages from an external (sometimes third-party) source. PyPI in the Python
+ecosystem would be an example of a remote repository in the context of your
+Artifactory instance.
+
+Phylum will serve as a remote repository, proxying requests to the actual package
+registry (e.g., PyPI, npm, etc.) while ensuring that packages that enter your
+organization’s Artifactory adhere to the policy you have defined regarding the
+acceptable use of open source libraries/packages.
+
+## Create a New Remote Repository
+
+1. Create a remote repository for Phylum. Navigate to `Repositories > Administration`
+   and click the green `Create a Repository` button in the top right and select `Remote`.
+
+   ![Artifactory_create_repo](../../assets/artifactory_create_repo.png)
+
+2. Select the appropriate package repository to which you would like to apply the
+   policy. [Phylum supported ecosystems](./about.md#supported-ecosystems).
+
+   ![Artifactory_select_ecosystem](../../assets/artifactory_select_ecosystem.png)
+
+3. Provide a name for your remote repository in the `Repository Key` field.
+4. In the `User Name` field enter the name of the Phylum group where you
```

Comment:
"Phylum group" and maybe "defined your policy" (or maybe not since it was linked earlier in this page) could use links to internal documentation.